plugin/src/sbt-test/scalafmt-sbt/sbt/build.sbt

```diff
@@ -69,12 +79,12 @@ lazy val p16 = project.settings(
 )
 lazy val p17 = project.settings(
   scalaVersion := "2.12.20",
-  TaskKey[Unit](
+  InputKey[Unit](
```

Comment:
what's the difference and significance to sbt2?

Reply:
so this is not a requirement for sbt2 but an optimization, and could have been submitted as a separate, unrelated PR?